### DISCUSSION TITLE
What LLM or AI coding tool works best with quasar

### DISCUSSION THREAD

**Discussion by etewiah** [quasarframework/quasar#17668] [General - Components / Directives / etc]
I've seen quite a few tools like bolt.new, v0 and replit that will generate code from text prompts.  They mostly seem to use react and tailwind.  Does anyone know if there are any tools like that which work well with quasar?

**Comment by samikrc** [comment]
I recently saw a video of someone using Windsurf editor to create a react app with shadcn/ui component. Should be close enough to try with quasar I am guessing.
https://www.youtube.com/watch?v=7gA2IouD-XU
